### ISSUE TITLE
Is there a pre-trained model for quickly test?

### ISSUE THREAD

**Issue by HymEric** [SilvioGiancola/SoccerNet-code#10]

**Comment by juvien** [comment]
this project did you complete?

**Comment by SilvioGiancola** [comment]
You can find [here](https://drive.google.com/open?id=1vc6NP7WCjwu-0tBRPUkzR0t_zYvO9Hm2) a copy of the pre-trained model that performed best in the SoccerNet paper.

Note that a novel state-of-the-art has been reached in this [CVPR2020](https://arxiv.org/pdf/1912.01326.pdf) paper, with [code to be released soon](https://github.com/cioppaanthony/context-aware-loss).

**Comment by maximelhoustau** [comment]
Hi Silvio,

I tried to use your pre-trained model with my own dataset, specifically on a single video. However when I try to load the models with `saver = tf.train.Saver().restore(sess, args.model)` in the ClassificationSecondBased.py file, it raises the following error:
`InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [64] rhs shape= [512]`

I am trying to make a prediction with ResNet features, on a video of 1 minute duration. Features have already been extracted, with a shape of (121, 2048). Would you have any suggestion to make your model work?

**Comment by SilvioGiancola** [comment]
Hi @maximelhoustau, Have you tried to use the PCA-reduced features for ResNet? Those should be vectors of dimension 512 per image.
